docs/rc.xml.all

```diff
@@ -78,10 +78,11 @@
     <raiseOnFocus>no</raiseOnFocus>
   </focus>
 
-  <!-- Set range to 0 to disable window snapping completely -->
   <snapping>
+    <!-- Set range to 0 to disable window snapping completely -->
     <range>1</range>
     <topMaximize>yes</topMaximize>
+    <notifyClient>both</notifyClient
```

Comment:
missing `>` at the end.

Random thought:
```xml
<notifyClient>always</notifyClient>
```
rather than `both` ?

Reply:
Good suggestions. Adopted (and syntax fixed) in the latest version.